test/fixtures/students.yml

```diff
@@ -1,6 +1,7 @@
 # Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 
 one:
+
```

Comment:
In the future, try not to add blank lines in files you haven't otherwise changed. This makes it more difficult to see what significant changes you've made.